packages/babel-plugin-react-native-web/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Babel plugin for React Native for Web",
   "main": "index.js",
   "devDependencies": {
-    "babel-plugin-tester": "^5.0.0"
+    "@babel/core": "^7.1.6",
```

Comment:
Why is `@babel/core` needed here?

Reply:
Not sure if it's required, it was automatically added by https://github.com/babel/babel-upgrade

Reply:
Removed